.github/workflows/upload_all.yml

```diff
@@ -1,26 +1,129 @@
+        run: exit 1
+      - run: echo "Uploading ${{ github.ref }}"
+      - name: Set upload targets
+        run: |
+          if [[ '${{ github.event.inputs.targets }}' =~ ^(all|google-play)$ ]]; then
+              echo "Uploading to Google Play"
+              echo "POLYPOD_UPLOAD_GOOGLE_PLAY=true" >> $GITHUB_ENV
+          fi
+          if [[ '${{ github.event.inputs.targets }}' =~ ^(all|app-store)$ ]]; then
+              echo "Uploading to the App Store"
+              echo "POLYPOD_UPLOAD_APP_STORE=true" >> $GITHUB_ENV
+          fi
+      - uses: actions/checkout@v2
+
+      # API and features
+      - name: API and features - Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+      - name: API and features - Lint
+        run: ./build.js lint
+      - name: API and features - Build
+        env:
+          POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
+          POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
+        run: ./build.js
+      - name: API and features - Test
+        run: ./build.js test
+
+      # Android
+      - name: Android - Setup
+        uses: actions/setup-java@v1
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        with:
+          java-version: 11
+      - name: Android - Test
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        working-directory: platform/android
+        run: ./gradlew test
+
+      - name: Android - Write keystore file
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        working-directory: platform/android/app
+        env:
+          KEYSTORE: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE }}
+        run: 'echo -n "$KEYSTORE" | base64 --decode > keystore.tmp'
+
+      - name: Android - Build
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        working-directory: platform/android
+        env:
+          POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE: keystore.tmp
+          POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS }}
+          POLYPOD_ANDROID_UPLOAD_KEY_ALIAS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEY_ALIAS }}
+        run: ./gradlew assemble -Psign=true
+
+      - name: Android - Determine release name
+        if: env.POLYPOD_UPLOAD_GOOGLE_PLAY == 'true'
+        run: |
+          version_code=$(./getPackageInfo.sh /usr/local/lib/android/sdk/build-tools/30.0.0/aapt ./app/build/outputs/apk/release/app-release.apk versionCode)
+          version_name=$(./getPackageInfo.sh /usr/local/lib/android/sdk/build-tools/30.0.0/aapt ./app/build/outputs/apk/release/app-release.apk versionName)
+          branch=${GITHUB_REF#refs/heads/}
+          sha_short=${GITHUB_SHA::7}
+          echo "##[set-output name=release_name;]$(echo $version_code ($version_name) - ${branch}@${sha_short})"
```

Comment:
I'd forgotten about this funkiness, but seems it still works for setting steps.vars.outputs.release_name.